src/renderer/app/index.tsx

```diff
@@ -8,19 +8,19 @@ import { walletModel } from '@entities/wallet';
 import { networkModel } from '@entities/network';
 import { proxyModel } from '@entities/proxy';
 import { notificationModel } from '@entities/notification';
-import { balanceSubscriptionModel } from '@features/balances';
+import { balanceSubModel } from '@features/balances';
+import { proxiesModel } from '@features/proxies';
 import { assetsModel } from '@pages/Assets/Assets/model/assets-model';
 import './i18n';
 import './index.css';
 import './styles/theme/default.css';
-import { proxiesModel } from '@features/proxies';
 
 log.variables.version = process.env.VERSION;
 log.variables.env = process.env.NODE_ENV;
 log.transports.console.format = '{y}/{m}/{d} {h}:{i}:{s}.{ms} [{env}#{version}]-{processType} [{level}] > {text}';
 log.transports.console.useStyles = true;
 
-Object.assign(console, log.functions);
+// Object.assign(console, log.functions);
```

Comment:
Is this a forgotten comment or do we now get rid of this :) ?

Reply:
Right, fixed that